packages/client/components/JiraScopingSearchHistoryMenu.tsx

```diff
@@ -72,13 +98,25 @@ const JiraScopingSearchHistoryMenu = (props: Props) => {
         const projectFilters = projectKeyFilters
           .map((filter) => filter.slice(filter.indexOf(':') + 1))
           .join(', ')
+        const handleRemoveJiraSearchQueryClick = (event) => {
```

Comment:
nit: `event: React.MouseEvent`
since we're a legacy codebase we don't enforce `noImplicitAny`, but it's nice to provide a type when we can!

Reply:
if this is the last item in the list, let's close it for them automatically because there's nothing more for them to do with the open menu.

e.g. 
```ts
if (jiraSearchQueries.length > 1) {
  event.stopPropagation()
}
```

Reply:
@mattkrick right, I'll pay more attention to adding proper types 🤗 
Also, I've added closing the menu when there's nothing left. 👍